AD419.Jobs.Core/Services/SqlDataContext.cs

```diff
@@ -54,13 +54,14 @@ public async Task RollbackTransaction()
         _sqlTransaction = null;
     }
 
-    public async Task ExecuteNonQuery(string script)
+    public async Task ExecuteNonQuery(string script, int timeoutSeconds = 30)
     {
-        using var connection = await GetOpenConnection();
```

Comment:
Did you mean to remove the using?

Reply:
Yes. It was rolling back transactions because they didn't have a chance to be committed. The connection is disposed when the `SqlDataContext` is disposed.